services/web/server/src/simcore_service_webserver/projects/_groups_api.py

```diff
@@ -0,0 +1,217 @@
+import logging
+from datetime import datetime
+
+from aiohttp import web
+from models_library.products import ProductName
+from models_library.projects import ProjectID
+from models_library.users import GroupID, UserID
+from pydantic import BaseModel, parse_obj_as
+from simcore_service_webserver.projects.models import UserProjectAccessRights
+
+from ..users import api as users_api
+from . import _groups_db as projects_groups_db
+from ._groups_db import ProjectGroupGetDB
+from .db import APP_PROJECT_DBAPI, ProjectDBAPI
+from .exceptions import ProjectInvalidRightsError
+
+log = logging.getLogger(__name__)
```

Comment:
Suggestion: I think we typically use `_logger`, so it might make sense to allign to that